code/main.py

```diff
@@ -119,19 +119,20 @@ def run_boot():
 
     # Main loop (runs every tick) -------------------------------------------------
     while True:
+        # Boot code
+        boot_menu.run()
+        crt.draw()
+
         # Event code
         for event in pygame.event.get():
             if event.type == pygame.QUIT:
                 pygame.quit()
                 sys.exit()
-            if event.type == pygame.KEYDOWN or event.type == pygame.MOUSEBUTTONDOWN:
+            if boot_menu.boot_text.anim_completed and (event.type == pygame.KEYDOWN or event.type == pygame.MOUSEBUTTONDOWN):
```

Comment:
Players should have the ability to just skip the booting. After all, it's just like an intro cutscene. So you should leave this part of the code alone.